examples/jest/jest.bzl

```diff
@@ -19,3 +19,11 @@ def jest_test(name, srcs, deps, jest_config, **kwargs):
         templated_args = templated_args,
         **kwargs
     )
+
+    # This rule is used specifically to update snapshots via `bazel run`
```

Comment:
can we do anything in the message when the test fails so that users know this bazel run command exists, if they want to just update the snapshot file? I think that's the critical bit to make this as easy to use as `-u`

Reply:
natively from the jest side I think the only way to influence it is a custom test reporter so let me see

Reply:
@alexeagle done